### ISSUE TITLE
The training loss issue

### ISSUE THREAD

**Issue by lgq-gpu** [zju3dv/OnePose#66]
Hello, first of all, thank you for sharing your code. I have a question: I'm training your code using an RTX 3060 GPU, with 40 epochs, batch_size=8, and haven't changed anything else. Currently, my loss has been consistently above 0.1, approximately 0.133. Is this normal?
